### DISCUSSION TITLE
Get best metrics after call to trainer.fit without evaluating best model on val set again

### DISCUSSION THREAD

**Discussion by curiousgradient** [Lightning-AI/pytorch-lightning#9016] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hi. I have a `ModelCheckpoint` callback where I am monitoring the `val_acc` (validation set accuracy) with `mode=max`. After training my model using `trainer.fit`, I want to get the maximum value of `val_acc`. How can I get this value without re-evaluating my best model on the validation dataset? Thanks.

**Comment by curiousgradient** [comment, marked as answer]
Can be accessed with checkpoint_callback.best_model_score.
